### PR TITLE
Update v6.6.0 CHANGELOG.md

### PR DESCRIPTION
**Pull request by hao-nan-li**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+* assuredworkloads: added `HEALTHCARE_AND_LIFE_SCIENCES_CONTROLS` and `HEALTHCARE_AND_LIFE_SCIENCES_CONTROLS_WITH_US_SUPPORT` enum values to `compliance_regime` in the `google_assuredworkload_workload` resource ([#19714](https://github.com/hashicorp/terraform-provider-google/pull/19714))
+* compute: added `bgp_best_path_selection_mode `,`bgp_bps_always_compare_med` and `bgp_bps_inter_region_cost ` fields to `google_compute_network` resource ([#19708](https://github.com/hashicorp/terraform-provider-google/pull/19708))
+* compute: added `next_hop_origin `,`next_hop_med ` and `next_hop_inter_region_cost ` output fields to `google_compute_route` resource ([#19708](https://github.com/hashicorp/terraform-provider-google/pull/19708))
+* compute: added enum `STATEFUL_COOKIE_AFFINITY` and `strong_session_affinity_cookie` field to `google_compute_backend_service` and `google_compute_region_backend_service` resource ([#19665](https://github.com/hashicorp/terraform-provider-google/pull/19665))
+* compute: moved `TDX` instance option for `confidential_instance_type` in `google_compute_instance` from Beta to GA(ga) ([#19706](https://github.com/hashicorp/terraform-provider-google/pull/19706))
```
Comment (melinath) [inline CHANGELOG.md:15]:
```suggestion
* compute: moved `TDX` instance option for `confidential_instance_type` in `google_compute_instance` from Beta to GA ([#19706](https://github.com/hashicorp/terraform-provider-google/pull/19706))
```

This should also be updated in the release notes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,32 @@
...
+* compute: promoted `client_destination_port` and `instance` fields in `google_compute_region_network_endpoint` resource to GA ([#19684](https://github.com/hashicorp/terraform-provider-google/pull/19684))
```
Comment (melinath) [inline CHANGELOG.md:16]:
```suggestion
```

This was reverted; this should also be updated in the release notes.

**Review by melinath — CHANGES_REQUESTED** [review]
a couple small changes to request

**Comment by hao-nan-li** [comment]
Thanks for the suggestions, release note updated.

**Review by melinath — APPROVED** [review]

**Pull request merged**